src/ray/util/BUILD

```
@@ -302,6 +302,7 @@ ray_cc_library(
         ":scoped_dup2_wrapper",
         ":stream_redirection_options",
         ":util",
+        "//src/ray/util/internal:stream_redirection_utils",
```

Btw, another feedback on naming, there is no need to call everything "utils" since it's already under the util folder :)
```suggestion
        "//src/ray/util/internal:stream_redirection",
```

---

In fact, I would instead name this file: `stream_redirection_handle`

---

sounds good! updated